src/Radio/RadioGroup.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React, { PureComponent, Children, cloneElement } from 'react';
+import React, { Component, Children, cloneElement } from 'react';
```

Comment:
@rosskevin I'm changing the behavior as `RadioGroup` accepts a `children` property (`React.createElement` returns a new instance at each render/calls).